src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -3,116 +3,71 @@
     "description": "This document is to validate user-defined FV3 forecast config files",
     "type": "object",
     "properties": {
-        "resources": {
+        "platform": {
```

Comment:
The platform section should be included in a workflow-level section, not in a driver-level section. Each driver can then use the single platform section as a single source of information.